daft/logical/logical_plan.py

```diff
@@ -593,6 +593,32 @@ def rebuild(self) -> LogicalPlan:
         return GlobalLimit(input=self._children()[0].rebuild(), num=self._num)
 
 
+class LocalCount(UnaryNode):
+    def __init__(self, input: LogicalPlan) -> None:
+        schema = Schema([Field("count", ExpressionType.from_py_type(int))])
+        super().__init__(schema, partition_spec=input.partition_spec(), op_level=OpLevel.PARTITION)
+        self._register_child(input)
+
+    def __repr__(self) -> str:
+        return self._repr_helper()
+
+    def resource_request(self) -> ResourceRequest:
```

Comment:
You can just remove this now! We made a refactor recently that returns `None` for the resource_request by default on the LogicalPlan super type